stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+  close();
+};
+
+const buildMenu = (menu: string) => (close: () => void) => [
+  <MenuItem key={1} value={1} onClick={buildClickHandler(menu, close)}>
+    {menu} item #1
+  </MenuItem>,
+  <MenuItem key={2} value={2} onClick={buildClickHandler(menu, close)}>
+    {menu} item #2
+  </MenuItem>,
+  <MenuItem key={3} value={3} onClick={buildClickHandler(menu, close)}>
+    {menu} item #3
+  </MenuItem>,
+];
+
+export const full: Story<HeaderProps> = (args) => {
+  return (
+    <Header
+      appLogo={
+        <HeaderLogo
+          logo={
+            <svg
+              viewBox='0 0 16 16'
+              xmlns='http://www.w3.org/2000/svg'
+              aria-hidden='true'
+            >
+              <path d='m12.6 13.4c-1.2-1.5-2.1-3.1-2.4-5.5-2.7 3.9-4.6 4.2-5.7 2.4l-1.2 5.7h-2.2l3.5-14.1 1.8-.4c-.1.5-1.4 6.2.6 7 2 .7 4.6-8.5 4.6-8.5l2.2.4c-1.6 3.7-1.6 7.6 1.1 10.9l-2.3 2.1' />
+            </svg>
+          }
+          onClick={() => action('Clicked on the title')()}
+        >
+          Microstation
+        </HeaderLogo>
+      }
+      breadcrumbs={
+        <HeaderBreadcrumbs
+          items={[
+            <HeaderButton
+              key='project'
+              menuItems={buildMenu('Project')}
+              name={'Project A (Super Size Edition)'}
+              description={'YJC-2249'}
+              startIcon={<SvgNetwork />}
+            />,
+            <HeaderButton
+              key='iModel'
+              menuItems={buildMenu('IModel')}
+              name={'IModel B'}
+              startIcon={
+                <img src='https://itwinplatformcdn.azureedge.net/iTwinUI/stadium.png' />
+              }
+              isActive={true}
+            />,
+            <HeaderButton
+              key='version'
+              name={'Version C'}
+              onClick={() => action('Clicked on the Version')()}
+              startIcon={<SvgVersion />}
+            />,
+          ]}
+        />
+      }
+      actions={[
+        <IconButton
+          key='notif'
+          onClick={() => action('Clicked on the notification bell')()}
+          styleType='borderless'
+        >
+          <SvgNotification />
+        </IconButton>,
+        <DropdownMenu key='help' menuItems={buildMenu('Help')}>
+          <IconButton styleType='borderless'>
+            <SvgHelpCircularHollow />
+          </IconButton>
+        </DropdownMenu>,
+      ]}
+      userIcon={
+        <DropdownMenu menuItems={buildMenu('User')}>
+          <IconButton styleType='borderless'>
+            <UserIcon
+              size='medium'
+              abbreviation='TR'
+              backgroundColor={getUserColor('Terry Rivers')}
+              image={
+                <img src='https://itwinplatformcdn.azureedge.net/iTwinUI/user-placeholder.png' />
+              }
+              title='Terry Rivers'
+            />
+          </IconButton>
+        </DropdownMenu>
+      }
+      menuItems={buildMenu('More')}
+      {...args}
+    >
+      <div>
```

Comment:
Is this div needed?

Reply:
Kinda was when I was using the `Full.args = { children: ....}` format, probably not needed anymore.